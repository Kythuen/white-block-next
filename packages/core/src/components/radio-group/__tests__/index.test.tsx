import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import RadioGroup from '../radio-group.vue'
// import { useRadioGroup } from '../use'

describe('# RadioGroup', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount({
        render() {
          return <RadioGroup />
        }
      })
      // expect(wrapper.classes()).toEqual(['wb-radio-group'])
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  // describe('## Slots', () => {})

  // describe('## Events', () => {})

  // describe('## Props', () => {})

  // describe('## Composables', () => {})
})
