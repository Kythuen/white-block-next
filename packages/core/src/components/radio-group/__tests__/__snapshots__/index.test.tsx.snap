// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# RadioGroup > ## Base > ### default config 1`] = `
<div
  bg="$wb-radio-group-background"
  class="wb-radio-group wb-radio-group--base wb-radio-group--soft wb-radio-group--primary wb-radio-group--rectangle wb-radio-group--md"
  color="$wb-radio-group-text"
  p="$wb-radio-group-padding"
  ring="1 inset $wb-radio-group-border"
  rounded="$wb-radio-group-radius"
>
  <div
    flex="~ row nowrap"
    relative=""
  >
    
    
    <!--v-if-->
  </div>
</div>
`;
